src/classes/Timer.class.php

```diff
@@ -39,18 +46,18 @@ public function stop()
 
     /**
      * Calculate difference between start and end time of event and return in milli-seconds.
+     * @throws \Exception
      * @return number time difference in milliseconds between stat and end time of event
-     * @throws Exception, if either of or both  of start or stop method are not called before this method
      */
     public function result()
     {
         if (is_null($this->start_time))
         {
-            throw new Exception('Timer: start method not called !');
+            throw new \Exception('Timer: start method not called !');
```

Comment:
I'm not sure.  I can see the argument for creating a Tripod\Exceptions\TimerException, since it's something you might want to catch because it's not something to fail over.